osl/maxfilter/maxfilter.py

```diff
@@ -573,13 +576,21 @@ def main(argv=None):
             sys.exit('Trans file not found ({0})'.format(args.trans))
 
     print('Processing {0} files'.format(sum(good_fifs)))
+    if args.outdir == None:
+        args.outdir = 'None'
+    print('we came to 0')
```

Comment:
debugging message?

Reply:
Clumsy me

Reply:
fixed in https://github.com/OHBA-analysis/oslpy/pull/48/commits/600ab9032269cc4e12ed28c8d5171ba0112de2cb